docs/userguide/networking/get-started-overlay.md

```diff
@@ -314,11 +311,15 @@ to have external connectivity outside of their cluster.
 
 You can try starting a second network on your existing Swarm cluster using Docker Compose.
 
-1. Log into the Swarm master.
+1. Install Docker Compose. If you install on a Mac or Windows with Docker
```

Comment:
1. If you haven't already, install Docker Compose.
    
Compose is installed by default, you'd have to do a custom installation to deselect it. Pretty likely they have it already if they installed with Toolbox. I agree with the @thaJeztah  the extra detail isn't needed.